tests/sentry/relay/config/test_metric_extraction.py

```diff
@@ -244,7 +245,10 @@ def test_get_metric_extraction_config_multiple_alerts_above_max_limit(
         assert len(config["metrics"]) == 1
 
         out, _ = capfd.readouterr()
-        assert out.split(": ")[-1] == "Too many (2) on demand metric alerts for project bar\n"
+        assert out[0].split(": ")[-2:] == [
+            "Spec version 0",
```

Comment:
I want to include checking the spec version in the output for tighter control.
I do not want the number of specs for the default version to change unintentionally.